core/src/test/java/org/apache/metamodel/QueryPostprocessDataContextTest.java

```diff
@@ -60,6 +60,49 @@
     private final Table table1 = schema.getTableByName(TABLE_CONTRIBUTOR);
     private final Table table2 = schema.getTableByName(TABLE_ROLE);
 
+    public void testSchemaTraversalWithAliasTable() {
+        final MockUpdateableDataContext dc = new MockUpdateableDataContext();
+        
+        final Column column = dc.getColumnByQualifiedLabel("foo");
+        assertEquals("table", column.getTable().getName());
+    }
+    
```

Comment:
I'm a bit missing a test to check that the SYSTEM_PROPERTY_CREATE_DEFAULT_TABLE_ALIAS flag actually properly disables the alias table without breaking stuff.

A bit the same with contexts that uses super(false)... Just to make sure that everything is (and keeps being) consistent.

Reply:
Added those tests now :-)